### DISCUSSION TITLE
Problem fetching video

### DISCUSSION THREAD

**Discussion by EHoglid** [sorry-cypress/sorry-cypress#669] [Q&A]
When trying to fetch video to dashboard I get this error:
_Failed to load resource: the server responded with a status of 416 (Requested Range Not Satisfiable)_ 

If I pase the link directly in chrome the video will be downloaded.
http://172.21.2.43:9000/sorry-cypress/af0b626c-61ee-44ed-a817-e57dc8a4b4ca.mp4

![2022-10-11_16-37-38](https://user-images.githubusercontent.com/10262990/195123542-58683cd9-9c5d-467d-b46f-36403594e34e.png)

**Comment by ragnito** [comment]
Hello, the same problem occurs to me. Have you managed to solve the problem?
